### DISCUSSION TITLE
Katas en Github

### DISCUSSION THREAD

**Discussion by Jimdrer** [LaunchX-InnovaccionVirtual/CursoIntroPython#70] [Q&A] [answered]
Las katas se estaran subiendo al Github?

**Comment by MSuarez0** [comment]
Este viernes dirán las indicaciones para subir las katas y segun tendras una semana apara poderlas subir.

**Comment by Gushley110** [comment]
Comentó en el live que sí se deben subir a Github y nos mandará un form dónde tendremos que poner la URL de nuestro repositorio personal.

**Comment by ViktorVizcarra** [comment]
si se subiran a tu cuenta gthub despues mediante un formulario te pediran tu nombre y la ruta del repositorio deonde esatn tus katas resueltas...

**Comment by doguedogue** [comment, marked as answer]
1. Crea un repositorio en tu github
2. Ve subiendo tus katas resueltas

**Reply by FernandaOchoa** [reply]
Así es, el día viernes publicaré el forms para que me manden el enlace de su repositorio y poder revisarlo.
